src/main/java/Duke.java

```diff
@@ -1,3 +1,4 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
```

Comment:
Is this a constant? Consider using uppercase and 'final' to represent it as a constant